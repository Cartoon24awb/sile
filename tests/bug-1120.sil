\begin[papersize=a6]{document}
\nofolios
\font[family=Libertinus Serif]
\set[parameter=document.parskip,value=10pt]

X

\script{
  local ot = SILE.require("core/opentype-parser")
	local font = ot.parseFont(SILE.shaper:getFace({}))
	for _, v in pairs(font.names[5]) do version = v[1]; break end
	SILE.typesetter:typeset(version)
	SILE.call("par")
}
\end{document}
