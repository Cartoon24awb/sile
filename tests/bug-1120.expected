Set paper size 	297.6377985	419.5275636
Begin page
Mx 	34.8819
My 	27.4464
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	57 w=6.6000	(X)
My 	49.4464
Mx 	5.7200
Mx 	4.4700
Mx 	3.7200
Mx 	3.9000
Mx 	2.7100
Mx 	5.0400
Mx 	5.4200
T	55 a=6.5200 70 a=4.4700 83 a=3.7200 84 a=3.9000 74 a=2.7100 80 a=5.0400 79 a=5.4200	(Version)
Mx 	68.8872
T	23 15 26 w=11.5000	(6.9)
End page
Finish
